ibis-server/app/model/data_source.py

```diff
@@ -161,3 +163,10 @@ def get_trino_connection(info: TrinoConnectionInfo) -> BaseBackend:
             user=(info.user and info.user.get_secret_value()),
             password=(info.password and info.password.get_secret_value()),
         )
+
+    @staticmethod
+    def _escape_special_characters_for_odbc(value: str) -> str:
+        if ";" in value or value.startswith("{"):
+            return "{" + value.replace("}", "}}") + "}"
+        else:
+            return value
```

Comment:
Could we always add brackets? Maybe we don't need to check if it contains any illegal character? 🤔

Reply:
It seems to be okay.